.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - python setup.py build
```

Comment:
`setup.py build` is not needed. It's always implied by `setup.py install`

Reply:
alright.